docs/design-document/03-connect/12-chatgpt-connector.md

```diff
@@ -0,0 +1,123 @@
+# ChatGPT
+
+## ChatGPT Source Connector
+
+### Configuration
+
+Before using ChatGPT source connector, you need to configure the server.
+- Please configure `sourceEnable` to `true` in `/resource/server-config.yml` to enable source functionality.
+- Please configure the source connector in `/resource/source-config.yml`:
+  - `connectorConfig`
+    - `connectorName`, name of the connector.
+    - (required) `path`, path of the API.
+    - (required) `port`, port of the API.
+    - `idleTimeout`, idle TCP connection timeout in seconds. A connection will timeout and be closed if no data is received nor sent within the `idleTimeout` seconds. The default is 0, which means don't timeout.
```

Comment:
The default value is 30 in your code. Why "_The default is 0, which means don't timeout_"?

Reply:
I think it is more appropriate to default to not timeout